### PR TITLE
Miscellaneous editorial fixes (#52).

### LINKED ISSUE #52: Update references and fix/clarify various problems with language.
- [x] Fix respec warning for normative TTML2 reference in non-normative section.
- [x] Replace ambiguous references to TTML1 with stable references to specific editions.
- [x] Improve reference labeling consistency for all TTML, IMSC references.
- [x] Use local bibliographic entries for all references to allow greater control of entry content.
- [x] Improve specificity and applicability of profile identification in abstract.
- [x] Miscellaneous editorial improvements to language.
- [x] Add co-editor.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #52.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/tt-profile-registry/pull/55.html" title="Last updated on Feb 28, 2019, 3:02 PM UTC (443cd2a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/tt-profile-registry/55/acd0e10...443cd2a.html" title="Last updated on Feb 28, 2019, 3:02 PM UTC (443cd2a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,6 +63,80 @@
           }],
 
           localBiblio: {
+            "IMSC1.0": {
```
Comment (nigelmegitt) [inline index.html]: This is in Specref as [`[ttml-imsc1.0.1-20180424]`](https://www.specref.org/?q=ttml-imsc1.0.1-20180424) which can be used here instead of defining it locally. For conciseness it can also be aliased to `IMSC1.0` for local use, for example:

```es
"IMSC1.0": {
aliasOf: "ttml-imsc1.0.1-20180424"
},
```

However I would prefer to use `IMSC1.0.1` rather than `IMSC1.0` - I'd take views on it though because I can see arguments both ways - any thoughts @palemieux ?

File: index.html [hunk 2]
```diff
@@ -60,6 +63,80 @@
...
+              authors: [
+                  "Pierre-Anthony Lemieux"
+              ],
+              title: "TTML Profiles for Internet Media Subtitles and Captions 1.0.1 (IMSC1)",
+              href: "https://www.w3.org/TR/2018/REC-ttml-imsc1.0.1-20180424/",
+              publisher: "W3C",
+              status: "REC",
+              date: "24 April 2018"
+            },
+            "IMSC1.1": {
```
Comment (nigelmegitt) [inline index.html:69]: This is in Specref as [`[ttml-imsc1.1-20181108]`](https://www.specref.org/?q=ttml-imsc1.1-20181108) which should be aliased instead of a local definition.

File: index.html [hunk 3]
```diff
@@ -60,6 +63,80 @@
...
+              authors: [
+                  "Pierre-Anthony Lemieux"
+              ],
+              title: "TTML Profiles for Internet Media Subtitles and Captions 1.1",
+              href: "https://www.w3.org/TR/2018/REC-ttml-imsc1.1-20181108/",
+              publisher: "W3C",
+              status: "REC",
+              date: "08 November 2018"
+            },
+            "TTML1-1e": {
```
Comment (nigelmegitt) [inline index.html:72]: This is in Specref as [`[ttaf1-dfxp-20101118]`](https://www.specref.org/?q=ttaf1-dfxp-20101118) which should be aliased instead of a local definition.

File: index.html [hunk 4]
```diff
@@ -60,6 +63,80 @@
...
+              authors: [
+                  "Glenn Adams"
+              ],
+              title: "Timed Text Markup Language (TTML) 1.0",
+              href: "https://www.w3.org/TR/2010/REC-ttaf1-dfxp-20101118/",
+              publisher: "W3C",
+              status: "REC",
+              date: "18 November 2010"
+            },
+            "TTML1-2e": {
```
Comment (nigelmegitt) [inline index.html:75]: This is in Specref as [`[ttml1-20130924]`](https://www.specref.org/?q=ttml1-20130924) which should be aliased instead of a local definition.

File: index.html [hunk 5]
```diff
@@ -60,6 +63,80 @@
...
+              authors: [
+                  "Glenn Adams"
+              ],
+              title: "Timed Text Markup Language (TTML) 1.0 (2nd Edition)",
+              href: "https://www.w3.org/TR/2013/REC-ttml1-20130924/",
+              publisher: "W3C",
+              status: "REC",
+              date: "24 September 2013"
+            },
+            "TTML1-3e": {
```
Comment (nigelmegitt) [inline index.html:78]: This is in Specref as [`[ttml1-20181108]`](https://www.specref.org/?q=ttml1-20181108) which should be aliased instead of a local definition.

File: index.html [hunk 6]
```diff
@@ -60,6 +63,80 @@
...
+                  "Glenn Adams",
+                  "Pierre-Anthony Lemieux"
+              ],
+              title: "Timed Text Markup Language (TTML) 1.0 (3rd Edition)",
+              href: "https://www.w3.org/TR/2018/REC-ttml1-20181108/",
+              publisher: "W3C",
+              status: "REC",
+              date: "08 November 2018"
+            },
+            "TTML1-SDP-US": {
```
Comment (nigelmegitt) [inline index.html:81]: This is in Specref as [`[ttml10-sdp-us-20130205]`](https://www.specref.org/?q=ttml10-sdp-us-20130205) which should be aliased instead of a local definition.

File: index.html [hunk 7]
```diff
@@ -60,6 +63,80 @@
...
+                  "Monica Martin",
+                  "Sean Hayes"
+              ],
+              title: "TTML Simple Delivery Profile for Closed Captions (US)",
+              href: "https://www.w3.org/TR/2013/NOTE-ttml10-sdp-us-20130205/",
+              publisher: "W3C",
+              status: "WG-NOTE",
+              date: "05 February 2013"
+            },
+            "TTML2-1e": {
```
Comment (nigelmegitt) [inline index.html:84]: This is in Specref as [`[ttml2-20181108]`](https://www.specref.org/?q=ttml2-20181108) which should be aliased instead of a local definition.

File: index.html [hunk 8]
```diff
@@ -73,698 +150,734 @@
...
+      <p>This document defines
+      the <a href='#mediatype'><code>application/xml+ttml</code></a> media
+      type and provides a registry of compact (short) identifiers that may be
+      used to identify TTML processor profiles. A processor profile specifies
+      a set of capabilities that a processor must support in order to process
+      a document, where this profile may be defined in a specification
+      document, in
+      a <a href="https://www.w3.org/TR/ttml1/#vocabulary-profiles">TTML
+      Profile Definition Document</a>, inline within a TTML Document Instance,
+      or by using a combination of these methods. This document document only
```
Comment (nigelmegitt) [inline index.html]: s/document document/document

File: index.html [hunk 9]
```diff
@@ -73,698 +150,734 @@
...
-        supported by the processor, would allow a first-order determination to
-        be made about whether a resource may be processed. We say <i>first
-          order</i> here since during actual decoding of a TTML resource, the
-        processor profile declarations contained within the resource may result
-        in the resource being rejected.
-      </p>
+      <p>A MIME type and subtype
+        combination, <code>application/ttml+xml</code>, was initially
+        defined in [[TTML1-1e]], Appendix C, and subsequently updated
+        by [[TTML1-2e]] and [[TTML1-3e]] in order to identify the
```
Comment (nigelmegitt) [inline index.html]: We have not updated the media type registration based on TTML1-3e because _this_ document became the registration document in place of TTML1-2e.

File: index.html [hunk 10]
```diff
@@ -73,698 +150,734 @@
...
+        content type of TTML resources. In addition, TTML as well as
+        other W3C and non-W3C specifications define a number of
+        processor profiles which specify requirements on processors
+        that may decode and process a TTML document. This document
+        augments the definition of this MIME type by specifying a
```
Comment (nigelmegitt) [inline index.html]: This isn't right - the IANA [registration](https://www.iana.org/assignments/media-types/application/ttml+xml) points at this document, so the published version of this document (i.e. https://www.w3.org/TR/ttml-profile-registry/) _is_ the definition of the MIME type and therefore cannot augment it.

There is some ambiguity introduced by the lines below ([line 241](https://github.com/w3c/tt-profile-registry/pull/55/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR241) in this pull request right now) that say "This section updates the media type", which is a relative statement compared to the previous registration, but I suggest we leave that as is.

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
Mostly editorial comments, such as aliasing specs defined in Specref rather than redefining them, and a typo, but the language about the Purpose of the document needs editing - it isn't right as is.

File: index.html (hunk 1 above)
Comment (palemieux) [reply index.html]: > I would prefer to use IMSC1.0.1

+1

File: index.html (hunk 2 above)
Comment (skynavga) [reply index.html:69]: I am completely opposed to making any use of specref references for the simple reasons that (1) they can (and do) change from under us without notice, thus making any document that employs them unstable, and (2) they do not contain the same information that we employ in our primary TTML specifications. We can discuss this further in teleconference.

File: index.html (hunk 3 above)
Comment (skynavga) [reply index.html:72]: ditto

File: index.html (hunk 4 above)
Comment (skynavga) [reply index.html:75]: ditto

File: index.html (hunk 5 above)
Comment (skynavga) [reply index.html:78]: ditto

File: index.html (hunk 6 above)
Comment (skynavga) [reply index.html:81]: ditto

File: index.html (hunk 7 above)
Comment (skynavga) [reply index.html:84]: ditto

File: index.html (hunk 1 above)
Comment (skynavga) [reply index.html]: ok, will change; (I mean I will change to use IMSC1.0.1, not use an alias to a specref entry - see below);

File: index.html (hunk 10 above)
Comment (skynavga) [reply index.html]: we need to discuss this in teleconference; if we can't update the definition, then the `codecs` parameter will not have normative status

File: index.html (hunk 2 above)
Comment (nigelmegitt) [reply index.html:69]: Using Specref is normal practice in W3 specs. If you want to change the entries, or have visibility of proposed change, this is done at https://github.com/tobie/specref .

File: index.html (hunk 10 above)
Comment (nigelmegitt) [reply index.html]: sure, I'll add it to the agenda at w3c/ttwg#11 

Why do we need to update the definition?

File: index.html (hunk 8 above)
Comment (nigelmegitt) [reply index.html]: Fixed in 5483f8e

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html]: Fixed in 5483f8e

File: index.html (hunk 2 above)
Comment (skynavga) [reply index.html:69]: That does not fix the document stability issue. This is really a pure editorial issue in the domain of the editors. Our choice is to use inline definitions to avoid instability due to unexpected and unannounced changes in specref over which  we have no editorial control.

Comment (nigelmegitt) [reply index.html:69]: That's not correct: the document is stable with respect to W3C specifications because the W3C /TR space spec info provided by specref is sourced from https://www.w3.org/2002/01/tr-automation/tr.rdf . If you think the Specref information isn't correct then we need to raise an issue with the team who maintain tr.rdf (or the tool that updates it on spec publication).

Comment (skynavga) [reply index.html:69]: I disagree. The sprecref data is not under direct control of the editors of this document nor is under the process control of this group; therefore, it is unstable and unfixed. If you like, we can simply convert this document to HTML5 or XML Doc format and stop using respec, but I will not agree to use respec if it forces us to use specref data to resolve bibliographic data. I note once again that this decision is in the domain of the editors (absent formal TTWG policy to the contrary).

Comment (nigelmegitt) [reply index.html:69]: In my view consistency across W3C documents takes precedence over consistency across TTWG documents; since the data is provided by W3C, we should use it.  Furthermore, I've just made the change locally to switch all the W3C spec references into aliases and the only differences I can see are:

1. the dates do not consistently zero-pad the day part to a two digit number in the text description of the document being referenced.
2. for SDP-US the phrase "W3C Note" becomes "W3C Working Group note"

So there does not seem to be a substantive problem with changing this, and we could consider raising a query about the date format.

Since I've made the change, I'd be happy to push it directly if you'd withdraw your objection @skynavga ?

Comment (skynavga) [reply index.html:69]: No, I do not withdraw my objection. As you have shown, even your attempt to synchronize the content was unsuccessful. The larger issue for me is that you are personally attempting to impose editorial control on a document for which you are not an editor, and you are blocking progress on the part of the editor's work.

Comment (nigelmegitt) [reply index.html:69]: > As you have shown, even your attempt to synchronize the content was unsuccessful.

I drew the opposite conclusion from the minor nature of the deltas! Essentially, I thought my attempt was successful and that the differences were so small as to be not worth holding out for.

Regarding "editorial control" I disagree, but as you've noted before, Github spec repository pull requests and issues aren't a good place to discuss process.

File: index.html (hunk 10 above)
Comment (skynavga) [reply index.html]: Addressed in https://github.com/w3c/tt-profile-registry/pull/55/commits/abc547e4d41e27c4ce0de4477443efba6b826f26 per a previous telecon.

File: index.html (hunk 2 above)
Comment (skynavga) [reply index.html:69]: I have looked at specref.org. It is not affiliated with the W3C and appears to impose copyright status on the content that may be used for specification data. As such, and here I am speaking with both my editor and AC representative hats, I cannot accept content that is arbitrarily inserted into our working group documents for which we as a TTWG and the W3C itself has no control and may be in violation of IPR policies.

Comment (nigelmegitt) [reply index.html:69]: @plehegar can you please provide guidance about https://github.com/w3c/tt-profile-registry/pull/55#discussion_r251211105 ? Specref seems to be used endemically in W3C specs and that seems like a good thing to me, but @skynavga is raising an issue that it is not suitable for use in this way.

A couple of other points worth considering @skynavga :
1. When the spec gets published to /TR the references etc are frozen, i.e. it's the respec formatted output that is uploaded, not the version with the respec script running. This means that even if the reference changes in Specref it won't magically break/fix documents in /TR.
2. Specref cannot claim copyright on the data it is providing, which is clearly owned by whatever organisations are publishing the specifications themselves. The only way it might have copyright is if the information is deliberately incorrect, which I believe is never the case.

**Comment by skynavga** [comment]
@nigelmegitt merged from PR #59; please approve for merger to gh-pages

File: index.html [hunk 11]
```diff
@@ -73,698 +97,730 @@
   </head>
   <body>
     <section id="abstract">
-      <p>This document defines the <a href='#mediatype'><code>application/ttml+xml</code></a> media type and provides a registry of identified TTML processor profiles. A processor profile is an identified reference to a set of capabilities that a processor supports, which may be defined in a specification document, a <a href='https://www.w3.org/TR/ttml1/#vocabulary-profiles'>TTML Profile Definition Document</a> or both.</p>
+      <h2>Abstract</h2>
+      <p>This document defines
+      the <a href='#mediatype'><code>application/xml+ttml</code></a> media
```
Comment (nigelmegitt) [inline index.html]: This edit reverts a previous fix: it should be `ttml+xml`

File: index.html (hunk 9 above)
Comment (nigelmegitt) [reply index.html]: This comment not resolved yet.

File: index.html (hunk 10 above)
Comment (nigelmegitt) [reply index.html]: This doesn't look addressed to me. Suggest rewording to:

> After the original MIME type was registered, it was augmented to include a `codecs` parameter. This document includes the public register of compact (short) identifiers to be used with this parameter during the exchange of TTML document entities.

File: index.html [hunk 12]
```diff
@@ -73,698 +97,730 @@
...
-        interpreted in whole or in part as making statements about the
-        conformance of a TTML resource or the features of TTML (or other
-        external specifications) actually used in the resource.
-      </p>
+        profile defining fora.</p>
+      <div class="note">
+        <p>In the context of this registry, the term <i>profile</i>
+          means <i>processor profile</i>. It explicitly does not
+          mean <i>content profile</i>. That is, nothing about the use
+          of the <code>profile</code> parameter described here is
```
Comment (nigelmegitt) [inline index.html]: needs to mention `codecs` too, e.g. replace with:

```
of the <code>profile</code> or <code>codecs</code> parameters described here is
```

**Review by nigelmegitt — CHANGES_REQUESTED** [review]

**Comment by skynavga** [comment]
@nigelmegitt I have attempted to address your comments at https://github.com/w3c/tt-profile-registry/pull/55#pullrequestreview-206754013 (and make a few respec warning fixes); please take a look

File: index.html [hunk 13]
```diff
@@ -73,698 +97,710 @@
...
-      Steering Group (IESG) for review, approval, and registration with the Internet
-      Assigned Numbers Authority (IANA).</p>
-
+    <section id="mediatype" class='normative'>
+      <h2>Media Type Registration</h2>
+      <p>This section defines the media type "application/ttml+xml",
+        and, in conformance with
+        <a href="https://www.ietf.org/rfc/rfc4288.txt">BCP 13</a>
+        and <a href="http://www.w3.org/2002/06/registering-mediatype.html">W3CRegMedia</a>,
+        serves as the registration thereof with the Internet Assianged
```
Comment (nigelmegitt) [inline index.html]: typo: s/Assianged/Assigned

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
Sorry, spotted a typo, will fix in place and then approve.

**Review by nigelmegitt — APPROVED** [review]
Thanks @skynavga, looking good to me now.
